fhservices.cfg.j2

```diff
@@ -121,10 +126,18 @@ define service {
 
 define service {
        service_description Storage::Pod::Disk Usage
-       hosts {{ rhmap_router_dns }}
        check_command check_pod_disk_usage!80!90
        use generic-service
-
+       hostgroup_name core,mbaas
        notes One or more pods have volumes which are running out of disk.  You should clean up the files on this pod, if possible, or scale down/up the pod so that it has a fresh filesystem.
        contact_groups rhmapadmins
 }
+
+define service {
+       service_description Database::MongoDB::Health
+       check_command check_mongodb!mongodb-service-1
```

Comment:
@jessesarn This is failing on my vagrant machine as the mongo service is called `mongodb-1`, I redeployed with the latest fh-openshift template and it is still called `mongodb-1`. Does this need to be renamed?